### PR TITLE
Bump build_root version of CNO to 1.15

### PR DESCRIPTION
**Pull request by rcarrillocruz**

### CODE REVIEW

File: ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.14
```
Comment (aojea) [inline ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml:39]: it was announced in the mailing list that we should move to 1.15
```
[aos-devel] ACTION REQUIRED: Migrating OpenShift 4.6 to (a default) of golang 1.15
```

Comment (rcarrillocruz) [reply ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml]: done

**Comment by danwinship** [comment]
/lgtm
but
/hold
becuase CI is currently dead and it would be good to confirm that this actually doesn't break anything before it comes back

**Pull request merged**